.gitignore

```diff
@@ -20,3 +20,6 @@ tmp
 mkmf.log
 target
 ruby/ext/rust_example/Makefile
+ruby/ext/rust_example/rust_example.o
+ruby/ext/rust_example/rust_example.so
+ruby/ext/rust_example/librust_example.a
```

Comment:
This is where they get built by default, though. So you'd need to copy them over.

Really, this repo isn't for serious-grade code, so the fact that they're here just isn't a big deal.